pyscriptjs/src/components/pyscript.ts

```diff
@@ -77,11 +77,10 @@ export class PyScript extends BaseEvalElement {
             }
         }
 
+        this.appendChild(mainDiv);
         if (currentMode == 'edit') {
             // TODO: We need to build a plan for this
-            this.appendChild(mainDiv);
         } else {
-            this.appendChild(mainDiv);
             addToScriptsQueue(this);
```

Comment:
I don't think the delete here is correct since we'll basically not add the main output div to the page...

Reply:
I don't think I understand whta you mean. The current logic says "If in edit mode, then append child. Otherwise, append child". 

So since it's always triggered regardless, I moved the appending function outside of the if/else. 
Am I missing something?

Reply:
That's true.
I think we could merge this 🤔